### ISSUE TITLE
Problem with Menu

### ISSUE THREAD

**Issue by LeoFC97** [rocketlaunchr/electron-alert#3]
Setup: 
S.O: windows 10
electron : ^4.1.4
electron-alert: ^0.1.3

Import:
const Alert = require("electron-alert")

My code on index.js:

![image](https://user-images.githubusercontent.com/33994116/71122676-ea347d00-21bf-11ea-8722-065e1ef3096c.png)

When i click test i recive this exception:  TypeError: this.browserWindow.removeMenu is not a function at Alert.fire

**Comment by pjebs** [comment]
I need to update the docs. The API has changed due to an accident. In the meantime can you use an earlier version or adapt your code based on the source code.

**Comment by pjebs** [comment]
I will fix it up asap

**Comment by pjebs** [comment]
```
Release Notes for v5.0.0-beta.3
Breaking Changes
Upgraded to Chromium 73.0.3683.27. #16975
Features
Added win.removeMenu() to remove application menus instead of using win.setMenu(null). #16657
```


@LeoFC97 Is there any chance you can use version 5 of electron?

**Comment by pjebs** [comment]
@LeoFC97 Should now be resolved with v0.1.4.

Let me know if you still have issues with electron 4.
